tests/unit/test_gitlab_auth.py

```diff
@@ -1,7 +1,10 @@
+from argparse import Namespace
```

Comment:
nit: ... 😊

Reply:
Removed this part now as we only have `dict` for options.